### ISSUE TITLE
get-involved 2.0 Communication section

### ISSUE THREAD

**Issue by wsmwk** [thundernest/thunderbird-website#201]
Looking to revise the Communication section, which needs an update and could be shorter.  But I think we could also tighten and improve the rest of the page - perhaps in a separate ticket?   

Starting points for the Communication section:

1. Removing the forum and mailing list headings to make the list shorter, and also remove distinction, which probably isn't useful to most folks.
2. Remove wiki paragraph, "Also check the Communication Channels wiki page." 
3. Remove all mentions of newsgroups and google group (news.mozilla.org will eventually go away and Firefox will standardize on Discourse)
4. Remove the top level "Discourse" item above "Thunderbird".  Perhaps even remove the Thunderbird item?
5. Add Support Crew and Add-on Developers matrix channels.
5. Add https://thunderbird.topicbox.com/groups/e2ee
6. Add the e2ee matrix channel OpenPGP.
7. Should developer items be listed under a Developers heading in the bullet list so they are both separated and easier to find?
8. Is it time to kill the engineering mailing list?
9. https://thunderbird.topicbox.com/groups/ux is unmentioned

I'm sure I've missed some items.

**Comment by wsmwk** [comment]
Speaking of communication, https://github.com/mozilla-frontend-infra/codetribute/issues/442 also needs to be done

**Comment by wsmwk** [comment]
I think we can do the work of https://github.com/thundernest/thunderbird-website/issues/243 here. 

With topicbox we drop discourse, newsgroups, and mailing lists.  And we go down to one subheading, as previously mentioned.  Maybe "Discussion and Help"?  

SUMO should stay listed first IMO, but maybe change the name to "Thunderbird Support and Help Articles"?

Might it would be less work if we have a short description of several topicbox groups and provide only one link to topicbox ?

That addresses items 1-4, 6, 9, 10.

And leaves 5,7,8. Plus anything else you think we need?

**Comment by Sancus** [comment]
OK, so page here: https://www-stage.thunderbird.net/en-US/get-involved/

I reorganized the ordering, as I suspect communications are the first thing new people care about, and also the list is mostly-alphabetical now(yes... design should come before development strictly speaking, but I like it better this way).

All the numbers above are addressed other than 8. Considering the list now, I'm not sure we actually need a separate developers section anymore. There are only like 2 items that would go in it(addon and developers lists).... if we did reorganize it that way, we would probably want to get rid of the Matrix section as well, and just do two subheadings that mix the two, so developers would have both lists and matrix channels for developers, and the other section would have all the rest.

But honestly I think it's fine as-is.

Some remaining things:
- Should Planning be at the top under Announcements? It may not be clear that it's one of our primary lists as-is...maybe we should bold the most important lists? Or just link to Topicbox and only call out Announcements, Planning and Development or something like that.... not sure.
- Testers no longer has a list? I left it out, do you want to re-add it after that list is recreated?
- Should we really link to Mozillazine? It's not ours and we don't control the content there. There is also very little activity. It's pretty much just @wsmwk and a couple other people posting links to release notes from time to time. I have left it out for now.
- I linked to the support crew wiki article, because you cannot see the Support Crew group on Topicbox without user privileges, I don't think. At least I can't see it from a private browsing window, it just 404s. Do we even want this link here?

**Comment by chrisilias** [comment]
- The Matrix section refers to Riot. Riot is now Element - https://element.io/

**Comment by Sancus** [comment]
>     * The Matrix section refers to Riot. Riot is now Element - https://element.io/

Good point. [Fixed](https://www-stage.thunderbird.net/en-US/get-involved/#communication).

**Comment by wsmwk** [comment]
All your reductions/removals are good.  I will fix the support crew wiki so people have a point of contact to join.

I like the reordering.   On the placement of planning - I think it's OK where you have it because IMO people should browse the full list to see what interests them, so order shouldn't be important. But I don't have a strong opinion.  Modulo support being at the top.

"high-level discussion about Thunderbird" is crazy vague, so I would slice and dice from the full description  "Discuss plans, strategy, policy, governance; improvements, meta issues; promote open standards; status, announcements; organize work to move Thunderbird forward!" 

Add-ons Developer list has redundancies - maybe shorten to something like "For developers to get support in developing and creating add-ons."

Both bullets about Wiki can be dropped. With that, we should be able to somehow simplify the remaining items under Documentation.  @chrisilias - thoughts on how we could flesh this area about better for people who want to contribute to KB and other areas?

Under matrix you can add
* Thunderbird & Chill - Thunderbird community relaxing around the water cooler, and not talking about Thunderbird.
* there is a UX channel 
* is anything else missing that is public?

**Comment by wsmwk** [comment]
Development intro could be improved.  How about "Learn how to contribute code to Thunderbird with our developer documentation. The doc describes how to get the code, set any necessary configuration, build and rebuild, etc. It is a good read - don’t just skim it."

I agree about not needing a developer heading under communication

**Comment by Alecaddd** [comment]
There are some small layout issues.

https://github.com/thundernest/thunderbird-website/blob/7dad12a4f65b9b2a8e699f0b5cd7a6d3c5a06bdb/website/get-involved/index.html#L296

Remove the `pr-8 pl-8` classes and wrap this whole section inside another section with these classes:
```
<section class="pt-10 lg:pt-20 bg-calendar pr-8 pl-8">...</section>
```
This is necessary to have the light grey curve properly blend with the darker grey curve of the pre-footer.

Remove the `lg:pb-10` class from the `documentation` section.

**Comment by Sancus** [comment]
> There are some small layout issues.

I always forget about the curvy gray line.... So, 'bg-calendar' adds it, but now the question is: Should that be on the communications section at the top, or not? On the current live page, there's only one at the bottom, so I assume that's correct and I should remove it from https://github.com/thundernest/thunderbird-website/blob/master/website/get-involved/index.html#L66 ?

**Comment by Alecaddd** [comment]
Yeah, it can be removed from the top.
You can use the `bg-grey-lighter` class to have a block section with slightly different background, just to give a tiny visual separation, but nothing super important since that grey is super light on purpose.

Maybe we should rename the `bg-calendar` class with something more semantically correct :D

**Comment by Sancus** [comment]
> bg-grey-lighter

tbh, I have a great deal of difficulty distinguishing this from white, even on a calibrated display. It's barely barely there for me if I know it's there, but I don't think I'd be able to notice it otherwise. It's RGB(253,253,253) lol which is less than 1% off-white? Granted, my eyesight is a good amount worse(20/30) than normal and my particular type of eye problem means I have some difficulty with extremely low contrast gradients.

In any case I decided not to use that 

@wsmwk 
> Both bullets about Wiki can be dropped.

Are you saying you want me to remove the link to: https://wiki.mozilla.org/Thunderbird:Home ? I thought it was still important documentation for aspects of the Council and a few other topics??

**Comment by wsmwk** [comment]
You're right, https://wiki.mozilla.org/Thunderbird:Home#Governance is still there.  But 90% of the wiki is now irrelevant/outdated.  I suggest instead have a governance-specific bullet point to  https://wiki.mozilla.org/Modules/Thunderbird#Thunderbird_Council. I have added a link from modules to Thunderbird:Home#Governance

Also, arguably, "Thunderbird Project" on the module's page should now point to https://www.thunderbird.net/en-US/about/ or get-involved.

**Comment by wsmwk** [comment]
> Are you saying you want me to remove the link to: https://wiki.mozilla.org/Thunderbird:Home ? I thought it was still important documentation for aspects of the Council and a few other topics??

https://www.thunderbird.net/en-US/about/ has a Thunderbird Council section which references the modules item. So perhaps this is sufficient.

**Comment by wsmwk** [comment]
(feel like I am rambling on)

Status meetings is missing.  ISTM this is important and should be a bullet, with a link to https://wiki.mozilla.org/Thunderbird/StatusMeetings

**Comment by Sancus** [comment]
> 
> 
> (feel like I am rambling on)
> 
> Status meetings is missing. ISTM this is important and should be a bullet, with a link to https://wiki.mozilla.org/Thunderbird/StatusMeetings

If we do a direct bullet point for status meetings only, then it should probably be in the communications section... but it doesn't fit in any of the two existing sections so it probably needs its own.

I feel like if we want to retire mentions of the wiki, we should only do that after we've genuinely moved all important content off it, and it seems like we haven't completed that yet.

Perhaps it's time for a wiki on thunderbird.net, which is something we should have the resources to be able to do now.

**Comment by wsmwk** [comment]
Overall looks good.

Nobody contributes to the TB wiki, so  "MozillaWiki:Policies/Accounts" is definitely pointless. (it was a grand wish, I might have even originally argued for its inclusion)

I'm pretty aware of all that's in the main wiki page and immediate sub pages, since I have authored or am the current curator for much of the material. So I can pretty safely say it's not worth keeping the links to the main page - that's the only TB wiki reference I'm suggesting to remove.  What is there that is super important?

**Comment by Sancus** [comment]
I don't have a strong opinion, I just thought it was strange to not mention the main index when we link to the wiki several times. [I've removed the mentions of the main wiki page](https://www-stage.thunderbird.net/en-US/get-involved/#communication) and added a status meeting section. That leaves 4 links to the wiki on the page:

- https://wiki.mozilla.org/Thunderbird:Testing
- https://wiki.mozilla.org/Thunderbird:Bug_Queries
- https://wiki.mozilla.org/Thunderbird/tb-support-crew
- https://wiki.mozilla.org/Thunderbird/StatusMeetings

So we'll need replacements/alternatives for those pages to drop wiki mentions completely. They all seem to contain something relevant. Not planning to handle that in this issue though.

**Comment by wsmwk** [comment]
Correct, those are all legitimate references.  And FWIW there is no urgency I know of to move the content of those elsewhere.

**Comment by wsmwk** [comment]
gah!  Looking under " Documentation  Help make sure the answers are written down before the questions are asked."  ...

I discovered we don't mention our documentation/KB https://support.mozilla.org/en-US/products/thunderbird  Perhaps with something like "You can create and improve user help documentation in SUMO".

**Comment by wsmwk** [comment]
>  Perhaps with something like "You can create and improve user help documentation in SUMO".

That said, it's not clear to me why we would keep two subheadings under Documentation.  Needs some thought.

**Comment by Sancus** [comment]
I've made that last change and most of these changes are live or will be soon, but https://www-stage.thunderbird.net/en-US/get-involved/ correctly reflects the current state of this page.

This issue is really old and has so much back and forth about in it that I'm going to close it here. If there's a desire to do further modifications to this page, open a fresh issue with the specific changes in mind.
